packages/authentication/src/definitions.ts

```diff
@@ -73,6 +73,14 @@ export interface FirebaseAuthenticationPlugin {
    * @since 0.1.0
    */
   getIdToken(options?: GetIdTokenOptions): Promise<GetIdTokenResult>;
+  /**
+   * Checks if an incoming link is a sign-in with email link suitable for signInWithEmailLink.
+   *
+   * The link sent to the user's email address must be provided.
```

Comment:
Since the methods no longer share the interface (see other comment below), you can force the parameter and remove this comment.